table/merging_iterator.cc

```diff
@@ -8,8 +8,10 @@
 // found in the LICENSE file. See the AUTHORS file for names of contributors.
 
 #include "table/merging_iterator.h"
+
 #include <string>
 #include <vector>
+
```

Comment:
I suggest to change as less as possible. Otherwise it will be pain to merge upstream.